### DISCUSSION TITLE
What are system requirements for running real time SLAM with AirSim

### DISCUSSION THREAD

**Discussion by catproof** [microsoft/AirSim#3888] [Support Q&A]
## Question
I wish to use a Visual-Inertial SLAM algorithm (something like ORB-SLAM3) to estimate the position of the drone and control the drone using that estimate. I plan to do this using the ROS wrapper. Aside from the system requirements listed here: https://microsoft.github.io/AirSim/faq.html#what-computer-do-you-need
How much extra computing power would I need to run a visual SLAM algorithm? I may also want to try running a depth estimation algorithm on top of all of that as well, to improve my localization.

**Comment by jonyMarino** [comment]
Hi @NickPerezCarletonUniversity and welcome!
I don't know the answer to that question. But I think it is a good idea to start with an azure VM and then adjust it until it meets your needs.
You can follow [this page](https://microsoft.github.io/AirSim/azure.html) if you can use a windows server with ROS over WSL2
